swift/vapor/main.swift

```diff
@@ -1,20 +0,0 @@
-import Vapor
-
-var services = Services.default()
-services.register(MiddlewareConfig())
```

Comment:
The equivalent of this would be:
```swift
app.middleware = .init()
```
Not having ErrorMiddleware will improve performance.